packages/react/src/components/Facepile/Facepile.test.tsx

```diff
@@ -181,49 +176,45 @@ describe('Facepile', () => {
       />,
     );
 
-    expectOne(wrapper, '.ms-Facepile-addButton .ms-Persona-coin.ms-Persona--size32');
-    expectOne(wrapper, '.ms-Facepile-overflowButton .ms-Persona--size32');
+    const [addButton, overflowButton] = getAllByRole('button');
 
-    const faces = findNodes(wrapper, '.ms-Facepile-person .ms-Persona-coin');
+    expect(addButton.querySelectorAll('.ms-Persona--size32')).toBeTruthy();
```

Comment:
querySelectorAll always returns a NodeList, so you should probably test the length or something instead